pkg/webview/view.proto

```diff
@@ -66,6 +66,8 @@ message K8sResourceInfo {
 
   // The span id for this pod's logs in the main logstore
   string span_id = 9;
+
+  repeated string k8s_display_names = 10;
```

Comment:
consider removing the `k8s` part of this, since this is already inside a K8sResourceInfo

Reply:
later on in this file, you have a YAMLResourceInfo. is the ultimate goal to get rid of that?

Reply:
YAMLResourceInfo and K8sResourceInfo are, sadly, two different things. Each resource is supposed to only have one a K8ResourceInfo, YAMLResourceInfo, LocalResourceInfo, etc

See here https://github.com/tilt-dev/tilt/blob/dmiller/ch6890/transfer-display-names-to-web-ui/internal/hud/view/view.go#L129:L129